sources/packages/web/src/router/InstitutionRoutes.ts

```diff
@@ -75,6 +76,15 @@ export const institutionRoutes: Array<RouteRecordRaw> = [
             },
             props: true,
           },
+          {
+            path: `${AppRoutes.LocationOfferings}/:locationId/education-program/:programId`,
```

Comment:
we can register the full URL in `AppRoutes.ts`

Reply:
Done